content/installation/installing-manager.md

```diff
@@ -46,7 +46,7 @@ We recommend that you specify an administrator password according to your securi
 ### Validating the Installation
 
 When the installation process is complete, you can use ```cfy status``` to make sure that all of the Cloudify services are running.
-Cloudify Premium customers can access the Cloudify Web Interface at: ```http://<manager_public_address>```
+Cloudify Premium customers can access the Cloudify Web Interface at: ```http(s)://<manager_public_address>```
```

Comment:
When is it http and when https?

Reply:
Depends on whether you have SSL enabled or not